assets/css/sponsori.css

```diff
@@ -0,0 +1,125 @@
+body{
+    margin: 0;
+}
+
+:root {
+    --black: white;
+    --white: black;
+    --secondary: rgb(237, 28, 36);
+    --secondary-transparent: rgba(237, 28, 35);
+  }
+
+h1{
+    font-size: 100px;
```

Comment:
This font size is too large. Let's stick with a maximum of 30 or 25px. Would be best if we would use `rem` for the `font-size` property